io.github.aerocyber.sitemarker.json

```diff
@@ -0,0 +1,41 @@
+{
+    "app-id" : "io.github.aerocyber.sitemarker",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "sitemarker",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=~/.sitemarker:create"
```

Comment:
To follow-up on the discussion from the old thread: you should update your app to respect XDG_CONFIG_HOME variable. This way you will be able to drop this line altogether.

Reply:
Alright, I will do that